.github/workflows/test-action.yml

```
@@ -31,3 +35,32 @@ jobs:
         else
           exit 0
         fi
+
+  test_chart_releaser_action:
+    runs-on: ubuntu-latest
+    permissions:
+      actions: none
+      checks: none
+      contents: none
+      deployments: none
+      issues: none
+      packages: none
+      pull-requests: none
+      repository-projects: none
+      security-events: none
+      statuses: none
+    name: Install chart-releaser and run it
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install chart-releaser
+      uses: ./
+      env:
+        CR_TOKEN: "FAKE_SECRETS"
+    - name: Check root directory
+      run: |
+        if [[ $(git diff --stat) != '' ]]; then
+          echo 'should be clean'
+          exit 1
+        else
+          exit 0
+        fi
```

You could use the `--exit-code` flag. The else branch is unnecessary.
```suggestion
        if ! git diff --stat --exit-code; then
          echo 'should be clean'
          exit 1
        fi
```